src/samples/visualiser/pygletExample.py

```diff
@@ -314,7 +314,7 @@ def __init__(self, width, height, x, y, target):
     parent = target.parent
     super().__init__(width, height, x, y, parent)
     self.camera = MapCamera(self.renderTarget, target.camera, 0.3)
-    self.skyboxTexture = pyglet.image.load("parchment.jpg").get_texture()
+    self.skyboxTexture = pyglet.image.load("src\\samples\\visualiser\\parchment.jpg").get_texture()
```

Comment:
```suggestion
    self.skyboxTexture = pyglet.image.load("src/samples/visualiser/parchment.jpg").get_texture()
```
Unsure if the alternative breaks linux compatibility